include/lsst/daf/base/DateTime.h

```diff
@@ -57,19 +59,27 @@ namespace base {
 
 class DateTime {
 public:
-    enum Timescale { TAI, UTC, TT };
-    enum DateSystem { JD, MJD, EPOCH };
+    enum DateSystem { JD=0, MJD, EPOCH };
+    enum Timescale { TAI=5, UTC, TT };  // use values that do not overlap DateSystem
+                                        // to avoid confusing one for the other in Python
```

Comment:
Does changing the start of the enum help with safety (I agree it makes it less confusing)? Wouldn't C++11 class enums be the way to go (I think swig supports them).